### PR TITLE
Added missing Apcu stub function "apcu_cache_info"

### PR DESCRIPTION
**Pull request by Geolim4**

### CODE REVIEW

File: apcu/apcu.php [hunk 1]
```diff
@@ -224,6 +224,16 @@ function apcu_cas($key, $old, $new){}
 function apcu_entry($key, callable $generator, $ttl = 0){}
 
 /**
+ * Retrieves cached information from APCu's data store
+ *
+ * http://php.net/manual/en/function.apcu-cache-info.php
```
Comment (neuro159) [inline apcu/apcu.php]: @link tag missing

**Comment by Geolim4** [comment]
Here you go :)

**Comment by Geolim4** [comment]
bump :)

**Comment by Geolim4** [comment]
@neuro159 bump :)

**Pull request merged**